env/AWSPW.env

```diff
@@ -34,81 +45,43 @@ if [[ "${step}" = "fcst" ]] || [[ "${step}" = "efcs" ]]; then
+    export APRUN_NP="${APRUN}"
 
     export NTHREADS_DWN=${nth_dwn:-1}
     [[ ${NTHREADS_DWN} -gt ${nth_max} ]] && export NTHREADS_DWN=${nth_max}
     export APRUN_DWN="${launcher} -n ${npe_dwn}"
 
 elif [[ "${step}" = "ecen" ]]; then
 
-    nth_max=$((npe_node_max / npe_node_ecen))
-
-    export NTHREADS_ECEN=${nth_ecen:-${nth_max}}
-    [[ ${NTHREADS_ECEN} -gt ${nth_max} ]] && export NTHREADS_ECEN=${nth_max}
-    export APRUN_ECEN="${launcher} -n ${npe_ecen}"
+    export NTHREADS_ECEN=${NTHREADSmax}
+    export APRUN_ECEN="${APRUN}"
 
     export NTHREADS_CHGRES=${nth_chgres:-12}
     [[ ${NTHREADS_CHGRES} -gt ${npe_node_max} ]] && export NTHREADS_CHGRES=${npe_node_max}
     export APRUN_CHGRES="time"
 
     export NTHREADS_CALCINC=${nth_calcinc:-1}
     [[ ${NTHREADS_CALCINC} -gt ${nth_max} ]] && export NTHREADS_CALCINC=${nth_max}
-    export APRUN_CALCINC="${launcher} -n ${npe_ecen}"
+    export APRUN_CALCINC="${APRUN}"
 
 elif [[ "${step}" = "esfc" ]]; then
 
-    nth_max=$((npe_node_max / npe_node_esfc))
-
-    export NTHREADS_ESFC=${nth_esfc:-${nth_max}}
-    [[ ${NTHREADS_ESFC} -gt ${nth_max} ]] && export NTHREADS_ESFC=${nth_max}
-    export APRUN_ESFC="${launcher} -n ${npe_esfc}"
+    export NTHREADS_ESFC=${NTHREADSmax}
+    export APRUN_ESFC="${APRUN}"
 
     export NTHREADS_CYCLE=${nth_cycle:-14}
     [[ ${NTHREADS_CYCLE} -gt ${npe_node_max} ]] && export NTHREADS_CYCLE=${npe_node_max}
-    export APRUN_CYCLE="${launcher} -n ${npe_esfc}"
+    export APRUN_CYCLE="${APRUN}"
 
 elif [[ "${step}" = "epos" ]]; then
 
-    nth_max=$((npe_node_max / npe_node_epos))
-
-    export NTHREADS_EPOS=${nth_epos:-${nth_max}}
-    [[ ${NTHREADS_EPOS} -gt ${nth_max} ]] && export NTHREADS_EPOS=${nth_max}
-    export APRUN_EPOS="${launcher} -n ${npe_epos}"
-
-elif [[ "${step}" = "postsnd" ]]; then
-
-    export CFP_MP="YES"
-
-    nth_max=$((npe_node_max / npe_node_postsnd))
-
-    export NTHREADS_POSTSND=${nth_postsnd:-1}
-    [[ ${NTHREADS_POSTSND} -gt ${nth_max} ]] && export NTHREADS_POSTSND=${nth_max}
-    export APRUN_POSTSND="${launcher} -n ${npe_postsnd}"
-
-    export NTHREADS_POSTSNDCFP=${nth_postsndcfp:-1}
-    [[ ${NTHREADS_POSTSNDCFP} -gt ${nth_max} ]] && export NTHREADS_POSTSNDCFP=${nth_max}
-    export APRUN_POSTSNDCFP="${launcher} -n ${npe_postsndcfp} ${mpmd_opt}"
-
-elif [[ "${step}" = "awips" ]]; then
-
-    nth_max=$((npe_node_max / npe_node_awips))
-
-    export NTHREADS_AWIPS=${nth_awips:-2}
-    [[ ${NTHREADS_AWIPS} -gt ${nth_max} ]] && export NTHREADS_AWIPS=${nth_max}
-    export APRUN_AWIPSCFP="${launcher} -n ${npe_awips} ${mpmd_opt}"
-
+    export NTHREADS_EPOS=${NTHREADSmax}
+    export APRUN_EPOS="${APRUN}"
 
 elif [[ "${step}" = "fit2obs" ]]; then
 
-    nth_max=$((npe_node_max / npe_node_fit2obs))
-
-    export NTHREADS_FIT2OBS=${nth_fit2obs:-1}
-    [[ ${NTHREADS_FIT2OBS} -gt ${nth_max} ]] && export NTHREADS_FIT2OBS=${nth_max}
-    export MPIRUN="${launcher} -n ${npe_fit2obs}"
+    export NTHREADS_FIT2OBS=${NTHREADS1}
```

Comment:
on the RHS is `NTHREADS1`.  Just checking of that is a typo or meant to be `${NTHREADS:-1}`

Reply:
Looking further, it seems `NTHREADS1=${nth:-1}`, so this is not a typo